OSDOCS-16939: DITA abstracts — OLM understanding topics (#115281)
openshift/openshift-docs: modules/olm-channel-schema.adoc

[id="olm-channel-schema_{context}"]
= olm.channel schema

[role="_abstract"]

> The `olm.channel` schema defines a channel within an Operator package, including bundle entries and upgrade paths for catalog indexes in {product-title}. Each entry names an `olm.bundle`, with optional `replaces`, `skips`, and `skipRange` fields that control upgrade graph behavior.

Reviewer: ```suggestion

The `olm.channel` schema defines a channel within a package, the bundle entries that are members of the channel, and the upgrade paths for those bundles.
```
